### PR TITLE
Added asset showing only the Big Dipper

### PR DESCRIPTION
**Pull request by micahnyc**
[bigdipper_constellation_data.zip](https://github.com/OpenSpace/OpenSpace/files/14845576/bigdipper_constellation_data.zip)

### CODE REVIEW

**Comment by micahnyc** [comment]
Closes #3047

File: data/assets/scene/milkyway/constellations/big_dipper.asset [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+asset.onDeinitialize(function()
+  openspace.removeSceneGraphNode(BigDipper)
+end)
+
+asset.export(BigDipper)
+
+asset.meta = {
+  Name = "Big Dipper",
+  Version = "1.1",
```
Comment (WeirdRubberDuck) [inline data/assets/scene/milkyway/constellations/big_dipper.asset:53]:
```suggestion
  Version = "1.0",
```

File: data/assets/scene/milkyway/constellations/big_dipper.asset [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+asset.onInitialize(function()
+  openspace.addSceneGraphNode(BigDipper)
+end)
+
+asset.onDeinitialize(function()
+  openspace.removeSceneGraphNode(BigDipper)
+end)
+
+asset.export(BigDipper)
+
```
Comment (WeirdRubberDuck) [inline data/assets/scene/milkyway/constellations/big_dipper.asset:50]:
```suggestion



```

File: data/assets/scene/milkyway/constellations/big_dipper.asset [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    DimInAtmosphere = true
+  },
+  Tag = { "daytime_hidden" },
+  GUI = {
+    Name = "Big Dipper",
+    Path = "/Milky Way/Constellations",
+    Description = [[Just the Big Dipper]]
+  }
+}
+
```
Comment (WeirdRubberDuck) [inline data/assets/scene/milkyway/constellations/big_dipper.asset:40]:
```suggestion


```

**Review by WeirdRubberDuck — APPROVED** [review]

File: data/assets/scene/milkyway/constellations/big_dipper.asset [hunk 4]
```diff
@@ -0,0 +1,58 @@
+local constellations_helper = asset.require("util/constellations_helper")
+
+
+
+local speck = asset.resource({
+  Name = "Big Dipper Constellation Files",
+  Type = "HttpSynchronization",
+  Identifier = "bigdipper_constellation_data",
```
Comment (alexanderbock) [inline data/assets/scene/milkyway/constellations/big_dipper.asset:8]:
```suggestion
  Identifier = "digitaluniverse_constellations_bigdipper",
```

File: data/assets/scene/milkyway/constellations/big_dipper.asset [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    NamesFile = speck .. "bigdipper.dat",
+    Colors = { { 0.6, 0.4, 0.4 }, { 0.8, 0.0, 0.0 }, { 0.0, 0.3, 0.8 } },
+    Unit = "pc",
+    DimInAtmosphere = true
+  },
+  Tag = { "daytime_hidden" },
+  GUI = {
+    Name = "Big Dipper",
+    Path = "/Milky Way/Constellations",
+    Description = [[Just the Big Dipper]]
```
Comment (alexanderbock) [inline data/assets/scene/milkyway/constellations/big_dipper.asset:37]: Should have a bit longer and more descriptive description.  Or alternatively take this away

**Review by alexanderbock — APPROVED** [review]

File: data/assets/scene/milkyway/constellations/big_dipper.asset [hunk 6]
```diff
@@ -0,0 +1,58 @@
+local constellations_helper = asset.require("util/constellations_helper")
```
Comment (alexanderbock) [inline data/assets/scene/milkyway/constellations/big_dipper.asset:1]:
```suggestion
```

**Pull request merged**